src/data/sql-quiz.ts

```diff
@@ -333,6 +333,17 @@ const sqlQuiz = [
     Explanation: "The DROP TABLE command is used to delete an entire table.",
     Link: "https://www.freecodecamp.org/news/learn-sql-queries-database-query-tutorial-for-beginners/",
   },
+  {
+    Question:
+      "In SQL, what is the command used to give a record a date and time value, from the system on execution?",
```

Comment:
I don't think we need this comma here

```suggestion
      "In SQL, what is the command used to give a record a date and time value from the system on execution?",
```